autogpts/autogpt/autogpt/agent_factory/generators.py

```diff
@@ -2,16 +2,16 @@
 
 from typing import TYPE_CHECKING
 
-from autogpt.config.ai_directives import AIDirectives
-from autogpt.file_storage.base import FileStorage
-
-from .configurators import _configure_agent
-from .profile_generator import generate_agent_profile_for_task
+from forge.config.ai_directives import AIDirectives
+from forge.file_storage.base import FileStorage
 
 if TYPE_CHECKING:
     from autogpt.agents.agent import Agent
-    from autogpt.config import Config
-    from autogpt.core.resource.model_providers.schema import ChatModelProvider
+    from forge.config.config import Config
+    from forge.llm.providers.schema import ChatModelProvider
+
+from .configurators import _configure_agent
+from .profile_generator import generate_agent_profile_for_task
```

Comment:
nit: Should these be fully qualified imports from autogpt.agent_factor.configurators import _configure_agent?

Reply:
IMO no. Imports from within the same module can be relative, which makes them easier to read and distinguish from dependencies on *other* modules.